From: Tobias Quathamer <toddy@debian.org>
Date: Wed, 29 Feb 2012 14:29:05 +0100
Subject: Correct search paths for documentation

---
 src/resource.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/resource.h b/src/resource.h
index 90a4dc1..f4949d7 100644
--- a/src/resource.h
+++ b/src/resource.h
@@ -103,8 +103,8 @@ const QByteArray EXPORT_FILTER   = QT_TRANSLATE_NOOP("file filter",
                                    + TEXT_FILTER);
 
 const QByteArray DATA_FILE       = "qbrewdata";
-const QByteArray HELP_FILE       = "handbook-index.html";
-const QByteArray PRIMER_FILE     = "primer.html";
+const QByteArray HELP_FILE       = "book/handbook-index.html";
+const QByteArray PRIMER_FILE     = "primer/primer.html";
 const QByteArray DEFAULT_FILE    = QT_TRANSLATE_NOOP("file name", "untitled");
 
 // previous valid data formats
-- 
